notifier/notifier.go

```diff
@@ -160,6 +180,13 @@ func (n *Notifier) ApplyConfig(conf *config.Config) error {
 		if err != nil {
 			return err
 		}
+
+		for _, am := range ams.ams {
```

Comment:
So I meant mainly my second comment on this thread :) The one about this not being needed here at all because the list of AMs is always empty at this point?